src/provider/model/Model.ts

```diff
@@ -87,7 +80,7 @@ export class Model {
         return Object.values(this._channelsById);
     }
 
-    public getWindow(identity: Identity): AppWindow|null {
+    public getWindow(identity: Identity): AppWindow | null {
```

Comment:
Is this linted? I think we enforce `a|b`

Reply:
Aren't we changing to `a | b` anyway with the new linting rules? I'll change it back since my VSCode auto formatted it.

Reply:
Maybe, but it will cause problems until then.